backup/key.go

```diff
@@ -0,0 +1,51 @@
+package backup
+
+import (
+	"fmt"
+	"github.com/coreos/go-semver/semver"
+	"path"
+)
+
+const (
+	// crd names for each provider
+	aws   = "aws"
+	azure = "azure"
+	kvm   = "kvm"
+
+	// namespace where secrets are located
+	secretNamespace = "default"
+
+	// namespace where crds are located
+	crdNamespace = "default"
+
+	fileMode = 0600
+)
+
+var awsSupportFrom *semver.Version = semver.Must(semver.NewVersion("4.0.0"))
+var azureSupportFrom *semver.Version = semver.Must(semver.NewVersion("1.0.0"))
```

Comment:
For now, we use some big future version. This will be set to proper value once we have changes in place.